core/src/test/java/org/mobilitydata/gtfsvalidator/validator/TableHeaderValidatorTest.java

```diff
@@ -16,8 +16,6 @@
 
```

Comment:
I would apply Google Java Format (static imports are before imports when doing so). Else than that, lgtm.

Reply:
Done. Thanks!